### PR TITLE
OZ-340: README as per Ozone's rebranding.

### PR DESCRIPTION
**Pull request by kdaud**
Ticket: https://mekomsolutions.atlassian.net/browse/OZ-340

### CODE REVIEW

File: maven-archetype/src/test/resources/projects/it-basic/reference/README.md
Comment (mks-d) [inline maven-archetype/src/test/resources/projects/it-basic/reference/README.md:1]: Can we remove this file altogether? I'm not sure why there was a README file within the test area of the project? 🤔

**Review by mks-d — CHANGES_REQUESTED** [review]
Thanks @kdaud! Just a comment about the second README file.

File: maven-archetype/src/test/resources/projects/it-basic/reference/README.md
Comment (kdaud) [reply maven-archetype/src/test/resources/projects/it-basic/reference/README.md:1]: It makes sense.

**Review by mks-d — APPROVED** [review]

File: maven-archetype/src/test/resources/projects/it-basic/reference/README.md
Comment (corneliouzbett) [reply maven-archetype/src/test/resources/projects/it-basic/reference/README.md:1]: Yes, this `README.md` file serves as a reference template for testing the generated child distribution "basicDistribution". The generated `README.md` is compared against this reference file to verify that the content matches correctly.

Comment (corneliouzbett) [reply maven-archetype/src/test/resources/projects/it-basic/reference/README.md:1]: Here is a PR to update & re-add it: https://github.com/ozone-his/ozone/pull/105

**Pull request merged**